runtime/services/vuid/watchdog.js

```diff
@@ -53,6 +53,8 @@ function startFeeding (timeout, callback) {
       return callback(err)
     }
     dog = yorkie
+    property.set('watchdog.pid', process.pid)
```

Comment:
Maybe the pid is not necessary for watchdog debugging, we could just listen the fd.

Reply:
Ok, we will remove it, and put the pid to shell script.